build/scripts/build.ps1

```diff
@@ -246,7 +248,7 @@ function Build-Artifacts() {
         Run-MSBuild "Compilers.sln" -useDotnetBuild
     }
     elseif ($build) {
-        Run-MSBuild "Roslyn.sln" "/p:DeployExtension=false"
+        Run-MSBuild "Roslyn.sln" "/p:DeployExtension=$deployExtensions"
```

Comment:
:bulb: it may be better to omit the property than set it to true. Setting it to true removes the ability of a project to unconditionally disable deployment via properties in the project file.